buble.rb

```diff
@@ -1 +1,39 @@
-puts "hello"
+def buble_sort(arr = [1, 0, 4, 3, 5, 1, 0, 1])
```

Comment:
- [ ] Remove the default arr in the argument,it should be like this
`def bubble_sort(arr)`
- [ ] Correct the spelling to `bubble_sort`